src/rebar3_format_prv.erl

```diff
@@ -81,7 +81,7 @@ get_files(Args, State) ->
                          "src/**/*.app.src",
                          "test/**/*.[he]rl",
                          "test/**/*.app.src",
-                         "*.config"];
+                         "{rebar,elvis,sys}.config"];
```

Comment:
@juanbono @diegomanuel I hope that you don't mind… but I would like the formatter by default *not* to mess up with `erlang_ls.config`. Is that ok with you?

Reply:
My goal was to avoid introducing `{format, [{ignore, ["erlang_ls.config"]}]}` in everyone of our rebar.configs.

Reply:
like our major repo does xD

Reply:
Yeah I think it is ok. Also it can mess up another config files :)